apps-rendering/src/components/Callout/calloutContact.tsx

```diff
@@ -52,12 +50,21 @@ const CalloutContact: FC<CalloutContactProps> = ({ format }) => {
 				css={calloutPrimaryButton(format)}
 				type="submit"
 				priority="primary"
-				icon={<SvgGps />}
 				href={OPEN_TELEGRAM_URL}
 				target="_blank"
 			>
 				Message us on Telegram
 			</LinkButton>
+
+			<LinkButton
+				css={calloutPrimaryButton(format)}
+				type="submit"
+				priority="primary"
+				href={OPEN_SIGNAL_URL}
+				target="_blank"
```

Comment:
`target="_blank"` can come with some consequences you may want to look into?

https://developer.chrome.com/docs/lighthouse/best-practices/external-anchors-use-rel-noopener/
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a#security_and_privacy